config/default.yml

```diff
@@ -317,3 +317,15 @@ Sorbet/ForbidComparableTEnum:
   Description: 'Disallows including the `Comparable` module in a `T::Enum`.'
   Enabled: true
   VersionAdded: 0.8.2
+
```

Comment:
Should this go in the `config/rbi.yml` instead?

Reply:
Oh I didn't know that existed 🤦🏻